redwood-lazylayout-compose/build.gradle

```
@@ -13,6 +13,7 @@ kotlin {
   sourceSets {
     commonMain {
       dependencies {
+        implementation libs.jetbrains.compose.runtime.saveable
```

```suggestion
        api libs.jetbrains.compose.runtime.saveable
```
This is exposed in public API